_episodes/01-introduction.md

```diff
@@ -69,15 +56,16 @@ teaching skills, but to connect instructors:
 
 > ## Key Readings
 >
-> The best overview of research results in education and learning is
-> Ambrose et al's *[How Learning Works][amazon-hlw]*,
-> which is also an excellent example of what secondary literature ought to look like.
```

Comment:
I'd like to keep this one - if you edit the file and push to your repo again, the pull request will automatically update. (The PR always shows the diff between the current state of what's in your repository, and what's in this one.)